metaflow/metaflow_config.py

```diff
@@ -403,6 +403,8 @@
 CONSOLE_TRACE_ENABLED = from_conf("CONSOLE_TRACE_ENABLED", False)
 # internal env used for preventing the tracing module from loading during Conda bootstrapping.
 DISABLE_TRACING = bool(os.environ.get("DISABLE_TRACING", False))
+MAX_MEMORY_PER_TASK = from_conf("MAX_MEMORY_PER_TASK")
```

Comment:
It would be better to have a default value because that is used to then convert env var values so we could set it to 0.0